src/application/InstanceManager.cpp

```diff
@@ -3,3 +3,4 @@
 const Repositories::EntryRepository* InstanceManager::entryRepository = nullptr;
 const Services::PasswordEncryptor* InstanceManager::passwordEncryptor = nullptr;
 const Services::PasswordDecryptor* InstanceManager::passwordDecryptor = nullptr;
+Services::PasswordGenerator* InstanceManager::passwordGenerator = nullptr;
```

Comment:
why is this not const?

Reply:
Because the state of the Mersenne Twister changes when generating numbers with it, which in turn changes the state of the Password Generator. Therefore it cannot be const since its state changes whenever a new password is requested.